examples/system_id_experiments/dataCollectorNorbot.cc

```diff
@@ -0,0 +1,81 @@
+#pragma once
```

Comment:
This doesn't really do anything (and seems to be missing a makefile) so is there any point in including it as an example?